--- xmp-scrobbler.orig.dev	2013-03-19 20:28:38.000000000 +0400
+++ xmp-scrobbler.dev	2013-03-19 21:35:02.000000000 +0400
@@ -3,16 +3,16 @@ FileName=xmp-scrobbler.dev
 Name=xmp-scrobbler
 UnitCount=14
 Type=3
-Ver=1
+Ver=2
 ObjFiles=
-Includes=
-Libs=
+Includes=C:\Dev-Cpp\MinGW32\include;C:\Dev-Cpp\include
+Libs=C:\Dev-Cpp\MinGW32\lib;C:\Dev-Cpp\lib
 PrivateResource=xmp-scrobbler_private.rc
 ResourceIncludes=
 MakeIncludes=
 Compiler=-DBUILDING_DLL=1_@@_-DCURL_STATICLIB_@@_
 CppCompiler=-DBUILDING_DLL=1_@@_-DCURL_STATICLIB_@@_
-Linker=--no-export-all-symbols --add-stdcall-alias_@@__@@_c:/Dev-Cpp/lib/libgdi32.a_@@_c:/Dev-Cpp/lib/libcurl.a_@@_c:/Dev-Cpp/lib/libssl.a_@@_c:/Dev-Cpp/lib/libssl32.a_@@_c:/Dev-Cpp/lib/libcrypto.a_@@_c:/Dev-Cpp/lib/libz.a_@@_c:/Dev-Cpp/lib/libwsock32.a_@@_c:/Dev-Cpp/lib/libwinmm.a _@@_
+Linker=--no-export-all-symbols --add-stdcall-alias_@@__@@_c:/Dev-Cpp/mingw32/lib/libgdi32.a_@@_c:/Dev-Cpp/lib/libcurl.a_@@_c:/Dev-Cpp/lib/libz.a_@@_c:/Dev-Cpp/mingw32/lib/libwsock32.a_@@_c:/Dev-Cpp/mingw32/lib/libwinmm.a _@@_
 IsCpp=1
 Icon=
 ExeOutput=
@@ -27,7 +27,9 @@ CustomMakefile=
 IncludeVersionInfo=0
 SupportXPThemes=0
 CompilerSet=0
-CompilerSettings=0000000000001000010100
+CompilerSettings=0000000000001000010101000
+LogOutput=
+LogOutputEnabled=0
 
 [Unit1]
 FileName=xmp-scrobbler.cpp
@@ -47,7 +49,7 @@ Build=1
 LanguageID=1033
 CharsetID=1252
 CompanyName=
-FileVersion=
+FileVersion=0.1.1.1
 FileDescription=Developed using the Dev-C++ IDE
 InternalName=
 LegalCopyright=
@@ -56,6 +58,7 @@ OriginalFilename=
 ProductName=
 ProductVersion=
 AutoIncBuildNr=0
+SyncProduct=0
 
 [Unit3]
 FileName=libscrobbler\errors.h
